docs/concepts/dev/inside.md

```diff
@@ -10,18 +10,24 @@ Hopsworks provides a Jupyter notebook development environment for programs writt
 
 Hopsworks provides source code control support using Git (GitHub, GitLab or BitBucket). You can securely checkout code into your project and commit and push updates to your code to your source code repository. 
 
-### Conda Environment per Project
+### FTI Pipeline Environments
 
-Hopsworks supports the self-service installation of Python libraries using PyPi, Conda, Wheel files, or GitHub URLs. The Python libraries are installed in a Conda environment linked with your project. Each project has a base Docker image and its custom conda environment. Jobs are run as Docker images, but they are compiled transparently for you when you update your Conda environment. That is, there is no need to write a Dockerfile, users install Python libraries in their project. You can setup custom development and production environments by creating new projects, each with their own conda environment.
+Hopsworks assumes that an ML system consists of three independently developed and operated ML pipelines.
```

Comment:
Somehow, "assumes" does not sound good to me. What about "Hopsworks postulates that building ML systems following the FTI pipeline architecture is best practice. This architecture consists of three independently developed and operated ML pipelines:"